Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home</title>
+    <style>
+      body {
+        background-color: #030759;
+        color: #6fa6a0;
+      }
+      a:link {
+        color: #f2d4c2;
+      }
+      a:visited {
+        color: #d9b8cd;
+      }
+    </style>
+  </head>
+  <body>
+    <div style="text-align: center">
+      <h1>Ambersoft</h1>
+      <br />
+      <a href="SourabhA3_2main.html">Home</a> |
+      <a href="SourabhA3_2mi11.html">MI-MEAN</a> |
+      <a href="SourabhA3_2contact.html">Contact</a>
+      <h1>Home Page</h1>
+      <br />
+    </div>
+    <br />
```

Comment:
<br> it is empty tag it doesn't have closing check it properly all br tag

Reply:
done